docs/src/markdown/extensions/tasklist.md

```diff
@@ -63,14 +63,28 @@ If `custom_checkbox` is enabled the structure will be as follows:
 </ul>
 ```
 
-Classes               | Description
---------------------- | ------------
-`task-list`           | Attached to either the `ul` or `ol` tag and represents the entire list element.
-`task-list-item`      | This is attached the `li` tag and represents an item in the list.
-`task-list-control`   | This is attached to the `label` tag and represents the control object.
-`task-list-indicator` | This is attached to the `span` directly following the input and is used to style the visual indicator.
+If `clickable_checkbox` is enabled checkboxes will be enabled:
 
-??? settings "Basic Tasklist CSS"
+```html
+<ul class="task-list">
+    <li class="task-list-item">
+        <label class="task-list-control">
+            <input type="checkbox" checked="">
+            <span class="task-list-indicator"></span>
+        </label>
+        item 1
+    </li>
+</ul>
+```
+
+| Classes               | Description                                                                                            |
+| --------------------- | ------------------------------------------------------------------------------------------------------ |
+| `task-list`           | Attached to either the `ul` or `ol` tag and represents the entire list element.                        |
+| `task-list-item`      | This is attached the `li` tag and represents an item in the list.                                      |
+| `task-list-control`   | This is attached to the `label` tag and represents the control object.                                 |
+| `task-list-indicator` | This is attached to the `span` directly following the input and is used to style the visual indicator. |
+
+### settings "Basic Tasklist CSS"
```

Comment:
This should be a details object, not a header.  Please change the syntax to how it was.